proposals/type-aliases.md

```diff
@@ -32,7 +59,7 @@ However, this would be possible with value types.
 Type aliases are declared using `typealias` keyword:
 ```
 typeAlias
-    : modifiers 'typealias' SimpleName (typeParameters typeConstraints)? '='
```

Comment:
I think we should mention explicitly in the text that constraints and variance are not allowed